### PR TITLE
reference/performance: add description for stats healthy

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Add a description for health statistics of tables

### What is the related PR or file link(s)? 

- Reference link(s):
- This PR is to align with: https://github.com/pingcap/docs-cn/pull/2097
- [ ] N/A (not applicable)

**Important Notice:**

**If your changes apply to multiple TiDB documentation versions**, to trigger the bot to cherry-pick this PR to other release branches, make sure you add labels such as:

- **needs-cherry-pick-3.0**
- **needs-cherry-pick-3.1**
- **needs-cherry-pick-2.1**

### CODE REVIEW

**Comment by juliezhang1112** [comment]
PTAL @lilin90 Thanks!

File: reference/performance/statistics.md [hunk 1]
```diff
@@ -185,6 +185,27 @@ Currently, the `SHOW STATS_META` statement returns the following 6 columns:
 >
 > When TiDB automatically updates the total number of rows and the number of modified rows according to DML statements, `update_time` is also updated. Therefore, `update_time` does not necessarily indicate the last time when the `ANALYZE` statement is executed.
 
+### Health statistics of tables
```
Comment (lilin90) [inline reference/performance/statistics.md]: "statistics" is a little bit weird to me, though it is used in the statement.

```suggestion
### Health state of tables
```

File: reference/performance/statistics.md [hunk 2]
```diff
@@ -185,6 +185,27 @@ Currently, the `SHOW STATS_META` statement returns the following 6 columns:
...
+
+You can use the `SHOW STATS_HEALTHY` statement to check the health statistics of tables and roughly estimate the accuracy of the statistics. When `modify_count` >= `row_count`, the health is 0; when `modify_count` < `row_count`, the health is (1 - `modify_count`/`row_count`) * 100.
```
Comment (lilin90) [inline reference/performance/statistics.md]:
```suggestion
You can use the `SHOW STATS_HEALTHY` statement to check the health state of tables and roughly estimate the accuracy of the statistics. When `modify_count` >= `row_count`, the health state is 0; when `modify_count` < `row_count`, the health state is (1 - `modify_count`/`row_count`) * 100.
```

**Comment by juliezhang1112** [comment]
Updated. PTAL again @lilin90 Thanks~

**Pull request merged**